packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/Choice.js

```diff
@@ -0,0 +1,105 @@
+import { Box, Button, Carousel, Heading, Paragraph } from 'grommet'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import { PrimaryButton, Media } from '@zooniverse/react-components'
+
+import Questions from './components/Questions'
+import checkFilledIn from './helpers/checkFilledIn'
+import getQuestionIds from './helpers/getQuestionIds'
+
+import counterpart from 'counterpart'
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+export default function Choice (props) {
+  const {
+    choiceId,
+    onCancel,
+    task
+  } = props
+  const {
+    choices,
+    images,
+    questions
+  } = task
+  
+  const [ answers, setAnswers ] = useState({})
+  console.log('answers', answers)
+
+  const choice = choices?.[choiceId]
+  const questionIds = getQuestionIds(choiceId, task)
+  const allowIdentify = checkFilledIn(answers, choiceId, task)
+
+  return (
+    <Box
+      flex='grow'
+      pad='small'
+    >
+      {choice.images.length > 0 && (
```

Comment:
Any chance of this being undefined? 
```suggestion
      {choice.images?.length > 0 && (
```